CHANGELOG.md

```diff
@@ -1,35 +1,39 @@
 # Cromwell Change Log
 
-## 29
+## 29 Release Notes
 
 ### Breaking Changes
 
-* Request timeouts for HTTP requests on the REST API now return a 503 status code instead of 500. The response for a request timeout is no longer in JSON format.
-* The metadata endpoint no longer returns gzipped responses by default. This now needs to be explicitly requested with an `Accept-Encoding: gzip` header 
+* **Command line**  
+In preparation for supporting CWL scripts (yes, you read that right!), we have extensively revised the Command Line in Cromwell 29. For more details about the usage changes please see the [README](https://github.com/broadinstitute/cromwell#command-line-usage). And stay tuned to the [WDL/Cromwell blog](https://software.broadinstitute.org/wdl/blog) over the next couple of months for more news about CWL.
 
-* Command line usage has been extensively revised for Cromwell 29.  Please see the
-[README](https://github.com/broadinstitute/cromwell#command-line-usage) for details.
+* **Request timeouts**   
+Cromwell now returns more specific `503 Service Unavailable` error codes on request timeouts, rather than the more generic `500 Internal Server Error`. The response for a request timeout will now be plain text, rather than a JSON format.
 
-* The engine endpoints are now served under `/engine`. Previousely engine endpoints were available under
-`/api/engine`. Workflow endpoints are still served under `/api/workflows`. The setting `api.routeUnwrapped` has been
-retired at the same time.
+* **Metadata endpoint**  
+The response from the metadata endpoint can be quite large depending on your workflow. You can now opt-in to have Cromwell gzip your metadata file, in order to reduce file size, by sending the `Accept-Encoding: gzip` header. The default behavior now does not gzip encode responses.
```

Comment:
I know I previously said "yeah" to this wording but now that I read it it sounds like the gzip'ing was new altogether. 

Can't think of the right place to put it but somehow indicating that the previous case was always giving you gzip'd data